### ISSUE TITLE
Create values.yaml with CCO CLI

### ISSUE THREAD

**Issue by zelima** [datopian/ckan-cloud-operator#74]
As a CCO user, I want to run 1 or few cco commands (maybe with prompts), and create values.yaml, so that I don't have to write by myself or copy from elswhere

As a CCO user, I want cco to have a default/general values.yaml to be created when deploying via helm, so that I don't need to spend time on searching what it is and create it by myself.

## Acceptance Criteria

- [ ] cco has command to create values.yaml
- [ ] cco uses some deful valyes.yaml if not defined

## Tasks

- [ ] Do analysis
- [ ] Refactor code

**Comment by pwalsh** [comment]
@akariv @zelima is this in a state for me to do acceptance testing yet?

**Comment by zelima** [comment]
No, values.yaml is still created manually for now afaik (@akariv correct me if I'm wrong )
